portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,22 +38,26 @@
   //constants for Datastore entity and property names
   private static final String ENTITY_COMMENT = "Comment";
   private static final String PROPERTY_CONTENT = "content";
+  private static final String PROPERTY_TIMESTAMP = "timestamp";
+  private static final String PROPERTY_COMMENT_NAME = "name";
```

Comment:
nit: for consistent with other fields, either name the constant PROPERTY_NAME, or name the other constants PROPERTY_COMMENT_CONTENT / PROPERTY_COMMENT_TIMESTAMP

Reply:
Got it! I'm changing them to be named PROPERTY_COMMENT_(name of property)